src/shared/record/record.ts

```diff
@@ -59,6 +77,16 @@ export class Record {
     return this;
   }
 
+  addImages(images: Image[]): Record {
+    this._images = this._images.concat(images);
```

Comment:
Should probably filter the ones it already has. See https://github.com/PronghornDigital/Montana-News-Archive/blob/master/src/shared/record/record.ts#L76